stablehlo/dialect/TypeInference.cpp

```diff
@@ -1337,22 +1337,19 @@ LogicalResult inferDynamicUpdateSliceOp(
   auto operandType = operand.getType().cast<ShapedType>();
   auto updateType = update.getType().cast<ShapedType>();
 
-  // (C3)
```

Comment:
Any reason for removing the constraints comment?

Reply:
Good catch!
I thought them to be stray comments. But realised that they are linked 1-1 to the spec constraints. Added them back.